api_v3/services/UserEntryService.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @service userEntry
+ * @package api
+ * @subpackage services
+ */
+class UserEntryService extends KalturaBaseService {
```

Comment:
All our services always implement add, update, get, delete and list, so please add get action.